code-challenges.test.js

```diff
@@ -22,29 +22,109 @@ const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "s
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
 
+describe("removeShuffle", () => {
```

Comment:
Excellent work on this section Nico!!  Your tests and logic look great!  To level up, try utilizing higher-order functions instead of for loops and consider edge cases for your tests.  Not a requirement, but just a way to stretch yourself even further.